### PR TITLE
[TW-1401] Doc saving collection runner order

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Add a new "Running your collection again" section
- Note that you can open a past run to run the collection again using the same custom request order

View the beta docs here: [Using the Collection Runner > Running collections again](https://tw-1401-save-run-order.learning.postman-beta.com/docs/collections/running-collections/intro-to-collection-runs/#running-the-collection-again)

### CODE REVIEW

**Review by postfrzero — APPROVED** [review]
LGTM!

File: src/pages/docs/collections/running-collections/intro-to-collection-runs.md [hunk 1]
```diff
@@ -111,6 +112,20 @@ To learn more about what happened during the collection run, do any of the follo
...
+
+After reviewing the results of the collection run, you can run the collection again. For example, you can edit the code for a failed test and run the collection again to check if the test succeeds.
+
+To run the collection again, do one of the following:
+
+* Select **Run Again** to run the collection again using the same settings.
+* Select **+ New Run** to configure a new run for the collection. Make changes to any settings, and then select **Run (collection name)** to run the collection again.
+* Select **Automate Run** to automate runs for the collection. You can [schedule a run](/docs/collections/running-collections/scheduling-collection-runs/), [run using the Postman CLI](/docs/postman-cli/postman-cli-run-collection/#running-a-collection-locally-with-the-postman-cli), or [integrate runs with your CI/CD pipeline](/docs/postman-cli/postman-cli-run-collection/#running-a-collection-in-cicd).
+
+> If you changed the selection and order of requests to run, the custom order isn't saved if you close or refresh the **Runner** tab or quit Postman.
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/collections/running-collections/intro-to-collection-runs.md]: @malvikach I've updated the note as follows:
> If you changed the selection or order of requests, or any other settings, the custom order and configuration are saved with the run results.
>
> To run the collection again later using your custom configuration, select the Runs tab in the collection to [view past runs](https://tw-1401-save-run-order.learning.postman-beta.com/docs/collections/running-collections/intro-to-collection-runs/#viewing-past-runs). Select the view report icon View report icon next to a run to open the collection run results. From here you can select Run Again to run the collection again using the same settings.

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

**Review by malvikach — APPROVED** [review]

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**